### ISSUE TITLE
Internal Server error

### ISSUE THREAD

**Issue by Anke** [markocupic/log-report-bundle#1]
Hallo Marco,

ich habe versucht, deine Extension unter C4.8.7 zu installieren. Das hat auch funktioniert, aber beim Aufruf der Einstellungen im BE bekam ich im Browser folgende Fehlermeldung:

```
Internal Server Error
Argument 2 passed to Contao\Controller::getTemplateGroup() must be of the type array, null given, called in /.../var/cache/prod/contao/dca/tl_settings.php on line 357
```

var/logs:
```
[2020-01-27 11:57:09] request.INFO: Matched route "contao_backend". {"route":"contao_backend","route_parameters":{"_route":"contao_backend","_scope":"backend","_token_check":true,"_controller":"Contao\\CoreBundle\\Controller\\BackendController::mainAction"},"request_uri":"[...]/contao?do=settings&ref=uNVkbaci","method":"GET"} []
[2020-01-27 11:57:09] request.CRITICAL: Uncaught PHP Exception TypeError: "Argument 2 passed to Contao\Controller::getTemplateGroup() must be of the type array, null given, called in /[...]/var/cache/prod/contao/dca/tl_settings.php on line 357" at /[...]/vendor/contao/core-bundle/src/Resources/contao/library/Contao/Controller.php line 87 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Argument 2 passed to Contao\\Controller::getTemplateGroup() must be of the type array, null given, called in /[...]/var/cache/prod/contao/dca/tl_settings.php on line 357 at /[...]/vendor/contao/core-bundle/src/Resources/contao/library/Contao/Controller.php:87)"} []
```
Was genau macht der Log-Report? Versand von Log-Berichten?

**Comment by markocupic** [comment]
Ja, genau. Du kannst auswählen, dass du bei best. Logeinträgen darüber informiert wirst.
Die Extension müsste wohl geupdated werden.
Lg Marko 
--
Diese Nachricht wurde von meinem Android Mobiltelefon mit GMX Mail gesendet.Am 27.01.20, 12:10 schrieb Anke <notifications@github.com>:

  Hallo Marco, 
  ich habe versucht, deine Extension unter C4.8.7 zu installieren. Das hat auch funktioniert, aber beim Aufruf der Einstellungen im BE bekam ich im Browser folgende Fehlermeldung: 
  Internal Server Error
Argument 2 passed to Contao\Controller::getTemplateGroup() must be of the type array, null given, called in /.../var/cache/prod/contao/dca/tl_settings.php on line 357
 
  var/logs: 
  [2020-01-27 11:57:09] request.INFO: Matched route "contao_backend". {"route":"contao_backend","route_parameters":{"_route":"contao_backend","_scope":"backend","_token_check":true,"_controller":"Contao\\CoreBundle\\Controller\\BackendController::mainAction"},"request_uri":"[...]/contao?do=settings&ref=uNVkbaci","method":"GET"} []
[2020-01-27 11:57:09] request.CRITICAL: Uncaught PHP Exception TypeError: "Argument 2 passed to Contao\Controller::getTemplateGroup() must be of the type array, null given, called in /[...]/var/cache/prod/contao/dca/tl_settings.php on line 357" at /[...]/vendor/contao/core-bundle/src/Resources/contao/library/Contao/Controller.php line 87 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Argument 2 passed to Contao\\Controller::getTemplateGroup() must be of the type array, null given, called in /[...]/var/cache/prod/contao/dca/tl_settings.php on line 357 at /[...]/vendor/contao/core-bundle/src/Resources/contao/library/Contao/Controller.php:87)"} []
 
  Was genau macht der Log-Report? Versand von Log-Berichten?
